src/backend/modules/auth/service/jwt.service.js

```diff
@@ -0,0 +1,23 @@
+const { sign, decode } = require('jsonwebtoken');
+const { JWT_SECRET, EXPIRE_DAYS } = require('../../../env');
+
+class Jwt {
+    secret;
+
+    expiresIn;
+
+    constructor() {
+        this.secret = JWT_SECRET;
+        this.expiresIn = EXPIRE_DAYS;
+    }
```

Comment:
Don't define it in constructor, constructor is for Dependencies Injection.
secret = JWT.SECRET, ...